fetcher-pipeline/gl_repo_urls.txt

```diff
@@ -6,7 +6,7 @@ https://content.bibletranslationtools.org/WA-Catalog/pt-br_ulb
 https://content.bibletranslationtools.org/WA-Catalog/my_ulb
 https://content.bibletranslationtools.org/WA-Catalog/ceb_ulb
 https://content.bibletranslationtools.org/WA-Catalog/zh_ulb
-https://content.bibletranslationtools.org/WA-Catalog/en_ulb
+https://content.bibletranslationtools.org/WycliffeAssociates/en_ulb
```

Comment:
@danparisd this requires deleting the current folder **en_ulb** in the docker volume (fetcher-repos) so that the new repo can be cloned to it